### PR TITLE
Update build to create example folders.

### PR DESCRIPTION
**Pull request by Helibobs**
On a windows system, the build process needs to create the example
sub folders, in order to progress.

Signed-off-by: helibobs helenm.666@hotmail.co.uk

### CODE REVIEW

File: make.py [hunk 1]
```diff
@@ -91,7 +91,9 @@ def make_example(viz, ex, i, count, fancy):
     extras = model['Extras'][:]
     example = 'example' + str(count)
     strdir = 'Examples/' + viz + '/'
-    
+    if not path.exists(strdir):
+		mkdir(strdir)
```
Comment (philogb) [inline make.py]: I think this should be indented.

**Comment by Helibobs** [comment]
Yes, I agree, but I am a little confused? Although the git hub viewer shows it as non-indented, it is actually indented in the committed file?  

https://github.com/Helibobs/jit/blob/auto_create_example_branches/make.py#L95.  

Having said that my change could be indented a little too much, after looking at it?
Thanks

**Comment by philogb** [comment]
Are you using tabs or spaces? I think that python files have 4-space
indentation.

On Wed, Aug 22, 2012 at 1:47 AM, Helibobs notifications@github.com wrote:

> Yes, I agree, but I am a little confused? Although the git hub viewer
> shows it as non-indented, it is actually indented in the committed file?
> 
> https://github.com/Helibobs/jit/blob/auto_create_example_branches/make.py#L95.
> 
> Having said that my change could be indented a little too much, after
> looking at it?
> Thanks
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/philogb/jit/pull/120#issuecomment-7928124.

## 

Nicolas Garcia Belmonte - http://philogb.github.com/

**Comment by Helibobs** [comment]
Apologies for that, not much of a python programmer.  Have amended the commit to use spaces.  Thanks.

**Comment by philogb** [comment]
Merged, thanks!

**Pull request closed without merging**